llvm/test/CodeGen/Hexagon/addrmode-opt-nonreaching.mir

```diff
@@ -0,0 +1,232 @@
+# It is not safe to make the transformation if the definition is killed by a call.
+# Use debug output for simplicity and test resilience.
+#
+# RUN: llc -march=hexagon -run-pass amode-opt %s -debug -o  %t_1.mir 2>&1 | FileCheck %s
+#CHECK: The LRExtReg reg {{.*}} is NOT live in to MBB
```

Comment:
Does `-passes='print<live-vars>'` provide a more specific/stable matching opportunity versus the debug output?

Reply:
Maintainers might be surprised if a change to the debug text wording causes a test failure here, for example.